src/pages/contributing/translation-program/contributors.tsx

```diff
@@ -51,11 +43,15 @@ export const getStaticProps = (async ({ locale }) => {
   }
 }) satisfies GetStaticProps<BasePageProps>
 
-const Content = (props: BoxProps) => (
-  <Box as={MainArticle} py={4} px={10} w="full" {...props} />
+const Content = ({ ...props }: BaseHTMLAttributes<HTMLHeadingElement>) => (
+  <div className="w-full px-10 py-4" {...props} />
 )
-const ContentHeading = (props: HeadingProps) => (
-  <OldHeading lineHeight={1.4} {...props} />
+
+const Text = ({
+  className,
+  ...props
+}: BaseHTMLAttributes<HTMLHeadingElement>) => (
+  <p className={cn("mb-[1.45rem] leading-[1.6rem]", className)} {...props} />
```

Comment:
I'll remove these custom leadings since we want to make this old page a bit more consistent with the other pages.